symplyphysics/laws/electricity/electrostatic_force_via_charges_and_distance.py

```diff
@@ -22,17 +22,17 @@
 
 first_charge = clone_as_symbol(symbols.charge, display_symbol="q_1")
 """
-First charge.
+First :symbols:`charge`.
 """
 
 second_charge = clone_as_symbol(symbols.charge, display_symbol="q_2")
 """
-Second charge.
+Second :symbols:`charge`.
 """
 
 distance = symbols.distance_to_origin
 """
-Distance between the charges.
+:symbols:`distance` between the charges.
```

Comment:
`distance_to_origin`
IMO it should be `length` here.

Reply:
Length is defined as the size of some object and not really the distance between two charges.

The problem I faced here is that `distance` is shown as `d` while I want it to be `r`. Therefore I opted for using `symbols.distance_to_origin` in place of `symbols.distance` for the actual symbol so that it would be shown as `r`, but used `` :symbols:`distance` `` in the description. And if I write `` :symbols:`distance_to_origin` `` the final text would read `distance_to_origin between the charges`, which is not what we want, I guess.

Reply:
It's a bad idea. We could use description like this: `Distance between the charges. See :symbols:distance_to_origin`. Another option is to use `d` in formula. It look more logical to have `d` for distance between charges, than `r`

Reply:
Fixed